composer.json

```diff
@@ -5,7 +5,7 @@
     "homepage": "https://airbrake.io",
     "license": "MIT",
     "require": {
-        "php": ">=5.4"
+        "php": "^5.4.7 || ^7.0"
```

Comment:
Can you please explain why for, example, version 5.4.1 is not supported? Why not `^5.4 || ^7.0`?